tasks/gen/templates/resource/operations/{{operation_filename}}.cr.ecr

```diff
@@ -1,5 +1,5 @@
 class <%= operation_class %> < <%= resource %>::SaveOperation
   # To save user provided params to the database, you must permit them
-  # https://luckyframework.org/guides/database/validating-saving#perma-permitting-columns
+  # https://luckyframework.org/guides/database/saving-records#perma-permitting-columns
```

Comment:
In PR: https://github.com/luckyframework/lucky/pull/1500
This file was renamed to save_{{underscored_resource}}.cr.ecr so after merging that PR the link in save_{{underscored_resource}}.cr.ecr will have to be updated.